app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryControls.js

```diff
@@ -54,6 +55,17 @@ class ExpandedStoryControls extends Component {
           type="button"
           value={I18n.t('cancel')}
         />
+
+        {
+          disabled && (
+            <ExpandedStoryToolTip text={I18n.t('accepted_tooltip')}>
+              <div className="infoToolTip">
+               <i className="mi md-18">info_outline</i>
```

Comment:
You missed a space here:
```suggestion
                <i className="mi md-18">info_outline</i>
```

Make sure that you have [EditorConfig](https://editorconfig.org/) installed in your text editor/IDE.